### PR TITLE
test: add some ut test in plugins numaaware policy

### PR DESCRIPTION
**Pull request by googs1025**
#### What type of PR is this?
unit test

#### What this PR does / why we need it:
- add some unit test for plugins/numaaware/policy
- fix some typo and error print
#### Which issue(s) this PR fixes:

See more detail https://github.com/volcano-sh/volcano/issues/3053#issuecomment-2042837878

#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
None
```

### CODE REVIEW

**Review by Thor-wl — COMMENTED** [review]
/lgtm

**Comment by lowang-bh** [comment]
As there are changes about uthelper lib, please wait other ut refacting PRs merged.

/hold

File: pkg/scheduler/plugins/numaaware/policy/policy_none_test.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2021 The Volcano Authors.
```
Comment (Monokaix) [inline pkg/scheduler/plugins/numaaware/policy/policy_none_test.go:2]: Plz rename to 2024.

Comment (googs1025) [reply pkg/scheduler/plugins/numaaware/policy/policy_none_test.go]: done. thank a lot

**Comment by Monokaix** [comment]
> As there are changes about uthelper lib, please wait other ut refacting PRs merged.
> 
> /hold

Seems that PRs have been merged: )

**Review by lowang-bh — APPROVED** [review]

**Pull request merged**